app/controllers/grade_entry_forms_controller.rb

```diff
@@ -47,6 +47,22 @@ def view_summary
     @date = params[:date]
   end
 
+  # returns the column distribution/breakdown for each grade_entry_item in a grade_entry_form
+  #   (see the second chart in the summary of a mark's spreadsheet, accessed from the dashboard)
+  def column_breakdown
+    @grade_entry_form = GradeEntryForm.find(params[:id])
```

Comment:
Because no views are being rendered, you don't need to worry about making this an instance attribute. So you can drop the `@` from the variable name.